### PR TITLE
[WIP] Add script to generate MD inputs

### PR DESCRIPTION
**Pull request by sidgoyal78**
don't merge

### CODE REVIEW

File: md_run_multiple.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+def get_system_id(trajname):
+    return int(trajname.split("/")[-1].split(".")[0][6:])
+
+
+def process_for_md(traj_name, opdir, index):
+
+    full_traj = ase.io.read(traj_name, ":")
+    sys_id = get_system_id(traj_name)
+
+    np.random.seed(sys_id) # seed the sampler
```
Comment (sidgoyal78) [inline md_run_multiple.py:23]: not required

File: md_run_multiple.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    VASP_FLAGS['laechg'] = False # don't save CHG
+
+    # adding the following from Muhammed's INCAR file
+    VASP_FLAGS['potim'] = 2
+    VASP_FLAGS['tebeg'] = 900 # 900 K
+    VASP_FLAGS['teend'] = 900 # 900 K
+    VASP_FLAGS['mdalgo'] = 0 
+    VASP_FLAGS['smass'] =  -3
+
```
Comment (mshuaibii) [inline md_run_multiple.py:42]: VASP_FLAGS['lcharg'] = False
(Forgot about this, wasn't included in my INCAR)

This actually controls whether CHG files get written. laechg controls the electron density files.

Comment (sidgoyal78) [reply md_run_multiple.py:42]: Done

**Comment by sidgoyal78** [comment]
Reviewed by @mshuaibii 
Input generation is in progress, closing this.

**Pull request closed without merging**